samples/android/app/src/main/java/org/mozilla/samples/glean/MainActivity.kt

```diff
@@ -45,6 +44,16 @@ open class MainActivity : AppCompatActivity() {
             )
         }
 
+        uploadSwitch.setOnCheckedChangeListener { _, isChecked ->
```

Comment:
We should also set the initial state of the toggle to match `getUploadEnabled()`